velox/exec/Spill.h

```diff
@@ -56,6 +56,10 @@ class SpillInput : public ByteStream {
 /// Represents a spill file that is first in write mode and then
 /// turns into a source of spilled RowVectors. Owns a file system file that
 /// contains the spilled data and is live for the duration of 'this'.
+/// NOTE: The class will not delete spill file upon destruction, so the user
```

Comment:
nit: leave an empty line between comment and note? Thanks!

Reply:
Then, they comment above would be logically detached from the class, which makes it harder to connect the comment and the class.